openapi/p4pa-organization.openapi.yaml

```
@@ -7,6 +7,32 @@ servers:
   - url: http://localhost:8080
     description: Generated server url
 paths:
+  /organization/apiKey/{organizationId}:
```

As first thing you identify the entity, then you will navigate its relationship or invoke an operation on it
```suggestion
  /organization/{organizationId}/apiKey:
```